components/company/EditForm.vue

```diff
@@ -0,0 +1,350 @@
+
+  <!-- Alert component -->
+  <VSnackbar v-model="alertState.show">
+    {{ $t(`${alertState.alertRoute}`) }}
+
+    <template #actions>
+      <VBtn
+        :color="alertState.type"
+        variant="text"
+        @click="alertState.show = false"
+      >
+        {{ $t('alert.close_alert') }}
+      </VBtn>
+    </template>
+  </VSnackbar>
+
+  <!-- Edit Form -->
+  <VForm ref="form" @submit.prevent="saveChanges || createCompany">
+    <VContainer>
+      <!-- Name -->
+      <VRow>
+        <FormTextInput
+          v-model="state.name"
+          :content="{
+            label: $t('edit.company.attributes.name'),
+          }"
+          :rules="[useRequiredInput]"
+        />
+      </VRow>
+
+      <!-- Description -->
+      <VRow>
+        <FormTextareaInput
+          v-model="state.description"
+          :content="{
+            label: $t('edit.company.attributes.description'),
+          }"
+        />
+      </VRow>
+
+      <!-- Company Type -->
+      <VRow>
+        <FormSelectInput
+          v-model="state.type"
+          :content="{
+            label: $t('edit.company.attributes.type.name'),
+            options: [
+              {
+                title: $t('edit.company.attributes.type.main_partner'),
+                value: 'main-partner',
+              },
+              {
+                title: $t('edit.company.attributes.type.partner'),
+                value: 'partner',
+              },
+              {
+                title: $t('edit.company.attributes.type.sponsor'),
+                value: 'sponsor',
+              },
+            ],
+          }"
+          :rules="[useRequiredInput]"
+        />
+      </VRow>
+
+      <!-- Webpage -->
+      <VRow>
+        <FormTextInput
+          v-model="state.webpage"
+          :content="{
+            label: $t('edit.company.attributes.webpage'),
+          }"
+          :rules="[useRequiredInput]"
+        />
+      </VRow>
+
+      <!-- Logo -->
+      <!-- TODO: #190 Add code for uploading image after company creation -->
+      <VRow>
+        <VCol :cols="state.logo ? 9 : 12">
+          <FormFileInput
+            :content="{
+              label: state.logo
+                ? `${state.logo.split('/').pop()}`
+                : $t('edit.company.attributes.logo'),
+              icon: 'mdi-image',
+              accept: ['image/png', 'image/jpeg'],
+            }"
+            :rules="[useRequiredInput]"
+            @change="setLogo"
+          />
+        </VCol>
+        <VCol v-if="state.logo && user?.userType === 'admin'" cols="3">
+          <VTooltip location="top" color="primary">
+            <!-- eslint-disable-next-line -->
+            <template #activator="{ props }">
+              <VBtn
+                v-bind="props"
+                variant="flat"
+                block
+                size="large"
+                rounded="lg"
+                color="error"
+                icon="mdi-delete"
+                @click="state.logo = null"
```

Comment:
I think it is a little unclear what this button does. It seems like it deletes the logo, but what it actually does is just clear the form input. I think the button should not be red and use a different icon to make this more clear.